appveyor.yml

```
@@ -43,4 +44,4 @@ for:
     skip_non_tags: true
     skip_tags: false
     build_script:
-      - ps: sh ./build-n-publish.sh --commit="$env:APPVEYOR_REPO_COMMIT" --version="$env:APPVEYOR_REPO_TAG_NAME" --platform=windows
+      - ps: sh ./build-n-publish.sh --image=$env:DOCKER_IMAGE_NAME --commit="$env:APPVEYOR_REPO_COMMIT" --version="$env:APPVEYOR_REPO_TAG_NAME" --platform=windows
```

What do you think about using quotes like with the other parameters?

```suggestion
      - ps: sh ./build-n-publish.sh --image="$env:DOCKER_IMAGE_NAME" --commit="$env:APPVEYOR_REPO_COMMIT" --version="$env:APPVEYOR_REPO_TAG_NAME" --platform=windows
```